coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanUtils.scala

```diff
@@ -93,7 +92,7 @@ object LogicalPlanUtils extends StrictLogging {
       case lp: LabelValues              => lp.copy(startMs = timeRange.startMs, endMs = timeRange.endMs)
       case lp: LabelNames               => lp.copy(startMs = timeRange.startMs, endMs = timeRange.endMs)
       case lp: LabelCardinality         => lp.copy(startMs = timeRange.startMs, endMs = timeRange.endMs)
-      case lp: TopkCardinalities        => lp.copy()
+      case lp: TsCardinalities          => lp.copy()
```

Comment:
No need to copy, we don't mutate the state anyways and there is no change in the input logical plan and output logical plan.

Reply:
Updated